src/main/java/com/iab/gdpr/consent/VendorConsent.java

```diff
@@ -21,15 +21,15 @@
     int getVersion();
 
     /**
-     * @return the {@link Instant} at which the consent string was created
+     * @return the {@link Date} at which the consent string was created
      */
-    Instant getConsentRecordCreated();
+    Date getConsentRecordCreated();
```

Comment:
Of course this in unavoidable, since Instant is not in Java 7. But now we are breaking API contract which means anybody who updates to latest version will probably have to change their code that uses this library. So we would need to bump the major version number for this release.

Reply:
Good point, bumped version to 3.0.0 in gradle file